server/lib/enrich/article.js

```diff
@@ -31,6 +31,13 @@ module.exports = exports = function article(content, format) {
 		content.bodyHTML = content.body;
 	}
 
+	content.hasGraphics = content.contentStats && content.contentStats.graphics && content.contentStats.graphics > 0 ? true : false;
+
+	const countOfGraphics = content.contentStats && content.contentStats.graphics && content.contentStats.graphics;
+	const countOfSharableGraphics = content.embeds ? content.embeds.filter(embed => embed && embed.type.endsWith('Graphic')).filter(embed => embed.canBeSyndicated === 'yes').length : 0;
```

Comment:
Instead of using `filter` twice, you could use one filter and have both conditions. 

Although the performance is not really an issue (the array is scanned twice with two filters) I think it's more readable to have the "predicate" of the filter to be one in this case. 

When will I use two filters? If the predicates where complicated enough to deserve to be separated